README.md

```diff
@@ -47,8 +51,13 @@ QIT currently supports the following types of tests:
 - [PHPStan Test](https://woocommerce.github.io/qit-documentation/#/test-types/phpstan)
 - [API Test](https://woocommerce.github.io/qit-documentation/#/test-types/api)
 - Compatibility Test
```

Comment:
Wonder if we should drop this or at least add a section on it to the documentation.

Reply:
Dropped it